### PR TITLE
Feat: added UI for profile page

### PR DESCRIPTION
**Pull request by Mohasin-Haque**
Added UI for profile page

image

### CODE REVIEW

**Review by chanchal-panpaliya — APPROVED** [review]
1) code looks clean 
2) proper naming 
3) good UI
4) good folder structure

File: src/App.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { Routes, Route } from "react-router-dom";
 import { HomePage } from "./pages/HomePage/HomePage";
 import { ArchivePage } from "./pages/ArchivePage/ArchivePage";
 import { TrashPage } from "./pages/TrashPage/TrashPage";
+import { ProfilePage } from "./pages/ProfilePage/ProfilePage";
```
Comment (akash-1318) [inline src/App.js]: You can use barrel imports here

**Review by akash-1318 — COMMENTED** [review]
Just added a simple suggestion besides that code is well written and no other changes require from my end .

File: src/App.js (hunk 1 above)
Comment (Mohasin-Haque) [reply src/App.js]: Done!!

**Pull request merged**